### DISCUSSION TITLE
Unable to resolve "wagmi/providers/public"

### DISCUSSION THREAD

**Discussion by ddrr1337** [WalletConnect/walletconnect-monorepo#4131] [Developer Support] [answered]
Hello, I am trying to use Web3Modal following the docs:
https://docs.walletconnect.com/web3modal/react-native/about

but I am getting this error:

 Unable to resolve "wagmi/providers/public" from "node_modules\@web3modal\wagmi-react-native\src\utils\defaultWagmiConfig.ts
 
 Any one has solved this?
 
 I am  using expo 49.0.10

**Comment by glitch-txs** [comment, marked as answer]
added the install version on docs :) thanks for reporting

```sh
npx expo install wagmi@1.4.13 viem@1.21.4
```
